gen_identity/gen_identity.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"fmt"
+	"github.com/spacemeshos/go-spacemesh/p2p"
+	"os"
+)
+
+func main() {
+
+	if len(os.Args) != 2 {
+		fmt.Printf("Usage: %v <path/to/output_dir>\n", os.Args[0])
+		return
+	}
+
+	key, err := p2p.EnsureIdentity(os.Args[1])
+	if err != nil {
+		fmt.Printf("failed generating identity: %v\n", err)
+	}
+
+	fmt.Printf("Private Key: %x\n", key)
```

Comment:
we need not the private key, but the id. check out in ensureIdentity how to get it. also, better to print only id without additional text